source/tutorial.ipynb

```diff
@@ -386,1731 +245,847 @@
-       "      <th id=\"T_15f33_level0_col1\" class=\"col_heading level0 col1\" >b</th>\n",
-       "    </tr>\n",
-       "  </thead>\n",
-       "  <tbody>\n",
-       "    <tr>\n",
-       "      <th id=\"T_15f33_level0_row0\" class=\"row_heading level0 row0\" >0</th>\n",
-       "      <td id=\"T_15f33_row0_col0\" class=\"data row0 col0\" >b</td>\n",
-       "      <td id=\"T_15f33_row0_col1\" class=\"data row0 col1\" >B</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n"
-      ],
-      "text/plain": [
-       "<pandas.io.formats.style.Styler at 0x26c5d89ea10>"
-      ]
-     },
-     "execution_count": 22,
-     "metadata": {},
-     "output_type": "execute_result"
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "start_time": "2023-08-30T03:55:39.366082Z"
     }
-   ],
+   },
+   "outputs": [],
    "source": [
-    "?[a, b] := *stored{l2: b, l1: a}"
-   ]
+    "?[a, b] := *stored[a, b]"
+   ],
+   "id": "1db0ba1654cbf140"
   },
   {
    "cell_type": "markdown",
-   "id": "135b9fb5-96fd-444b-aa44-ad8dd754318d",
-   "metadata": {},
+   "metadata": {
+    "ExecuteTime": {
+     "start_time": "2023-08-30T03:59:24.314463Z"
+    }
+   },
    "source": [
-    "If the binding is the same as the column name, you can omit the binding:"
-   ]
+    "Unlike relations defined inline, the columns of stored relations have fixed names. \n",
+    "You can take advantage of this by selectively referring to columns by name,\n",
+    "especially if you have a lot of columns:"
+   ],
+   "id": "6b193f51fe6acbfb"
   },
   {
    "cell_type": "code",
-   "execution_count": 23,
-   "id": "a5a29a9c-5320-4c9f-a153-a96c662cf5d6",
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/html": [
-       "<style type=\"text/css\">\n",
-       "#T_941fb_row0_col0 {\n",
-       "  color: black;\n",
-       "}\n",
-       "</style>\n",
-       "<table id=\"T_941fb\">\n",
-       "  <thead>\n",
-       "    <tr>\n",
-       "      <th class=\"blank level0\" >&nbsp;</th>\n",
-       "      <th id=\"T_941fb_level0_col0\" class=\"col_heading level0 col0\" >l2</th>\n",
-       "    </tr>\n",
-       "  </thead>\n",
-       "  <tbody>\n",
-       "    <tr>\n",
-       "      <th id=\"T_941fb_level0_row0\" class=\"row_heading level0 row0\" >0</th>\n",
-       "      <td id=\"T_941fb_row0_col0\" class=\"data row0 col0\" >B</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n"
-      ],
-      "text/plain": [
-       "<pandas.io.formats.style.Styler at 0x26c5d89fa50>"
-      ]
-     },
-     "execution_count": 23,
-     "metadata": {},
-     "output_type": "execute_result"
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "start_time": "2023-08-30T03:55:39.366665Z"
     }
-   ],
+   },
+   "outputs": [],
    "source": [
-    "?[l2] := *stored{l2}"
-   ]
+    "?[a, b] := *stored{l2: b, l1: a}"
```

Comment:
Newb to cozo, datalog and Jupyter here but I'm just working through the tutorial and I think this and similar places are wrong?

`stored` is defined above as `{c1, c2}` so I think this wants to reference `c1` and `c2` instead of `l1` and `l2`? At least that seems to make it happy in the wasm playground where I'm playing along...

I was going to submit a PR but I wasn't sure if this was the right file to edit or if this was generated from another file which needed editing?